inst/rmarkdown/homepage/index.Rmd

```diff
@@ -0,0 +1,363 @@
+Each protocol can either be accessed online through this website (html version) or a PDF version can be downloaded.
+On the left side you will find the navigation to the different sections.
+In the NEWS section you can find which (versions of) protocols were added when.
+
+This website contains the published protocols.
+The management, maintenance, approval and revision of the source files underlying these published protocols is done in the [inbo/protocolsource](https://github.com/inbo/protocolsource) GitHub repository.
+
+### Five types of protocols
+
+We distinguish five types of standard protocols.
+The first type is the standard field protocol (`sfp`) which is used to explain the steps that are needed in the field by a field worker to execute a specific field activity.
+The second type is the standard instrument protocol (`sip`) which explains the use and maintenance of instruments.
+The third type are the standard analytical protocols (`sap`) which are used in the laboratory or the field to explain a particular lab or field-analytical procedure involving physical or chemical measurements.
+The fourth type stands for standard operating procedures (`sop`) and is for everything that does not fit in one of the previous types.
+
+The fifth type is a category reserved for project-specific protocols (`spp`).
+The latter is more of a convenience to aid quick development and employment of protocols within a project context.
+Within the scope of a project, it might sometimes be necessary to re-use a `sfp` type protocol but with slightly different settings which are specific to the project.
+This can be done by including a published `sfp` protocol as an `spp` subprotocol where the deviations from the default settings are given.
+
+
+### Protocol identification
+
+The previously explained typology of protocols is further detailed in a so-called protocol-code.
+A **protocol-code** consists of a prefix referring to the *protocol-type* (three characters), a *protocol-number* (three digits) and a *language tag* (two characters), all separated by a hyphen.
+
+| type       | theme      | theme_number | protocol-code |
+|:-----------|:-----------|:-------------|:--------------|
+| field      | generic    | 0            | sfp-0##-en    |
+| field      | water      | 1            | sfp-1##-en    |
+| field      | air        | 2            | sfp-2##-en    |
+| field      | soil       | 3            | sfp-3##-en    |
+| field      | vegetation | 4            | sfp-4##-en    |
+| field      | species    | 5            | sfp-5##-en    |
+| instrument |            |              | sip-###-en    |
+| operating  |            |              | sop-###-en    |
+| analysis   |            |              | sap-###-en    |
+| project    |            |              | spp-###-en    |
+
+The three digits of the protocol-code will be referred to as the *protocol-number*.
```

Comment:
Repetition of previous sentence, see remark in Dutck.

Reply:
Done in [13b6571](https://github.com/inbo/protocolhelper/pull/97/commits/13b65714278609708268f5625505a54d5939d37f)